src/templates/detail/main.html

```diff
@@ -246,11 +245,58 @@
 {% except %}
 {% end %}
 
-{% include "detail/content_ratings.html" %}
-
-<section class="main full">
-  <a class="button support" href="{{ url('app/abuse', [slug]) }}">
-    {{ _('Report Abuse') }}</a>
+<!--More information (detail/more-info.styl).-->
+<section class="main full app-more-info">
+  <div>
+    {% defer (url=endpoint, as='app', key=slug) %}
+      <div class="detail-flex-wrap">
+        <div class="detail-left-side">
+          <h3>{{ _('More Information') }}</h3>
+          <dl class="app-more-metadata">
+            <div>
```

Comment:
Putting `<div>`s inside of a `<dl>` seems wrong. [MDN suggests](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl) that `<dl>` should only have `<dt>` and `<dd>` elements in it. Is it possible to remove the `<div>`s? I do like that this is in a `<dl>`.